codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/InputTypeGETQueryItemMiddleware.kt

```diff
@@ -48,7 +50,7 @@ class InputTypeGETQueryItemMiddleware(
     private fun renderApplyBody() {
         for (member in inputShape.members()) {
             val memberName = ctx.symbolProvider.toMemberName(member)
-            val queryKey = member.memberName
+            val queryKey = member.getTrait<HttpQueryTrait>()?.value ?: member.memberName
```

Comment:
For the query key/name, use the value in the query trait if one is available, else just use member name as before.